app.js

```diff
@@ -25,10 +28,19 @@ app.get('/impersonations/current', async function (req, res, next) {
   const data = {
     type: 'impersonations',
     id: sessionId,
+    attributes: {
+      'original-session-roles': originalSessionRoles,
+    },
     relationships: {
       impersonates: {
         links: `/resources/${impersonatedResourceId}`,
         data: { type: 'resources', id: impersonatedResourceId },
+      },
+      'original-resource': {
+        data: { type: 'resources', id: originalResourceId }
```

Comment:
If we stay in the non-generic space, we could rename this to `original-account` to make it more clear.